atd-vzv/src/views/map/helpers.js

```diff
@@ -1,3 +1,5 @@
+import { mapRequestFields } from "../summary/queries/socrataQueries";
```

Comment:
I'm curious if you benchmarked to see how much this helped the payload. Did it have much of an effect?

Reply:
Good call. Yes, even though the additional years of data increased the payload overall, filtering down the columns returned reduced the payload quite a bit.

#### Before (current year data only)
<img width="1002" alt="Screen Shot 2020-08-10 at 12 28 05 PM" src="https://user-images.githubusercontent.com/37249039/89812989-9be53d00-db06-11ea-9187-fbe64e0256ea.png">

#### After (without selecting columns)
<img width="1002" alt="Screen Shot 2020-08-10 at 12 26 32 PM" src="https://user-images.githubusercontent.com/37249039/89813032-a99ac280-db06-11ea-8890-70615d5537fa.png">

#### After (selecting only necessary columns)
<img width="1002" alt="Screen Shot 2020-08-10 at 12 25 25 PM" src="https://user-images.githubusercontent.com/37249039/89813061-b4edee00-db06-11ea-994c-6651b97ff829.png">